lametro/utils.py

```diff
@@ -106,3 +109,38 @@ def get_list_from_csv(filename):
         my_list = [row for row in reader]
 
     return my_list
+
+
+class LAMetroRequestTimeoutException(Exception):
+    pass
+
+
+def get_url(url, params=None, **kwargs):
+    """
+    Convenience function to ensure GET requests that time out raise an exception.
+
+    See https://stackoverflow.com/a/71453648
+    """
+
+    TOTAL_TIMEOUT = kwargs.get("timeout", 5)
```

Comment:
What do you think about making this configurable (i.e., defining it in settings.py)? We can set a reasonable default there.

Reply:
Yeah, that's a good call.